builds/ZIKA.md

```diff
@@ -1,5 +1,10 @@
 # ZIKA Pipeline Notes
 
+## Setup
+
+1. Start fauna rethinkdb server in a separate terminal.
+2. Make sure environment variables for connecting to fauna are set.
```

Comment:
Maybe worth pointing out in the README here that 1 is not relevant to our own builds since there's a central fauna server?

Reply:
Huh, I just tried it and the pipeline actually does work without running `rethinkdb` in a separate terminal. Thanks! I was doing using the combination of `rethinkdb` and `chateau` to view the changed database. Yes, I vote for dropping step 1.

Reply:
#117